### PR TITLE
Show param1 and param2 in debug screen

### LINKED ISSUE #5920: Show more infos in debug screen
I'd like to see more infos in the debug screen:

* About the pointed node:
    * [ ] `param1` **(rejected by paramat)**
    * [X] `param2`
    * [ ] Has metadata y/n
    * [ ] Texture of pointed face instead of top texture **(rejected by paramat)**
    * [ ] “Real” name (`descripton`), if present (maybe shorten it, if too long) **(rejected by paramat)**
* About the pointed entity:
    * [ ] Entity type ID (e.g. `__builtin:falling_node`, `boats:boat`, …)
* [ ] Mapgen name
* [ ] Heat and humidity point of current position (if applicable)
* [ ] Current biome (if applicable)

Especially `param2` is very important for mod development.
If the text becomes too long, we might consider adding a 3rd line.

Feel free to suggest other information. :-)

### PR DESCRIPTION
**Pull request by Wuzzy2**
* Show `param1` and `param2` in debug screen
* Add units to debug screen (ms, s, °)
* Make texts slightly more readable

Partially fixes #5920.

### CODE REVIEW

File: src/game.cpp [hunk 1]
```diff
@@ -4303,17 +4303,17 @@ void Game::updateGui(const RunStats &stats, f32 dtime, const CameraOrientation &
 		std::ostringstream os(std::ios_base::binary);
 		os << std::fixed
 		   << PROJECT_NAME_C " " << g_version_hash
-		   << " FPS = " << fps
-		   << " (R: range_all=" << draw_control->range_all << ")"
+		   << "; " << fps << " FPS"
```
Comment (sfan5) [inline src/game.cpp]: for consistency i would just prefer this to stay at "FPS = 123" instead of "123 FPS"

**Comment by Wuzzy2** [comment]
Hey! Why has param1 been removed out of the debug screen again? >:-(

**Comment by SmallJoker** [comment]
Removed in d4cc49e
Related discussion: http://irc.minetest.net/minetest-dev/2017-08-11#i_5040751

**Comment by nerzhul** [comment]
i think it can be a good idea to make debug screen being a CSM mod, when we can create labels on screen

**Comment by paramat** [comment]
I used to think param1 (light) was useful, i supported showing this information, but actually it is 0 for all solid nodes, and unreadable for any light source because it is encoded together with daylight using lower and upper 4 bits. So almost completely useless.

**Comment by Wuzzy2** [comment]
OK.

**Pull request merged**